factory/cert_utils.go

```diff
@@ -82,12 +85,22 @@ func NewSignedCert(signer crypto.Signer, caCert *x509.Certificate, caKey crypto.
 		return nil, err
 	}
 
+	expirationDays := defaultNewSignedCertExpirationDays
+	envExpirationDays := os.Getenv("CATTLE_NEW_SIGNED_CERT_EXPIRATION_DAYS")
```

Comment:
@nickgerace The original issue ([and the comment](https://github.com/rancher/rancher/issues/35068#issuecomment-959838243))  says we're updating the expiry duration to 10 years instead of one, so where is this env var being set to 10y?

Reply:
I will be doing that in another PR, likely to webhook itself! This is the library it uses, but good question. I should have clarified that.